lib/diagnostic.rb

```diff
@@ -28,18 +33,24 @@ def example
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
+    it 'gets /examples' do
+      get '/examples'
+
+      expect(response).to be_success
+    end
 end
```

Comment:
Great. We'd also just want to make sure the response we're getting is what we'd expect, with some combo of:

```

    examples_response = JSON.parse(response.body)
    expect(examples_response.length).to eq(examples.count)
    expect(examples_response['examples'].first['title']).to eq(example.title)
```